e2e/tests/components/dashboard/dashboard.e2e-spec.ts

```diff
@@ -579,4 +479,117 @@ describe('Dashboard Tests', () => {
 
         expect(await imageCompareFullPageScreen('dashboard-stacked-mode-widget-order')).toEqual(0);
     });
+
+    describe('Stacked Mode', () => {
+
+        beforeEach(async () => {
+            // resize page so in stacked mode
+            await browser.driver.manage().window().setSize(400, 600);
+        });
+
+        it('should allow widgets to be moved while in stacked mode', async () => {
+            await browser.actions().dragAndDrop(widget2, { x: 0, y: -250 }).perform();
+
+            expect(await page.getWidgetLocationValue(widget2, 'top')).toBe(0, 'widget2 top');
+            expect(await page.getWidgetLocationValue(widget2, 'left')).toBe(0, 'widget2 left');
+
+            const expectedLayout = [
+                { id: 'analytics-1-widget', col: 0, row: 1, colSpan: 4, rowSpan: 2 },
+                { id: 'subscription-widget', col: 0, row: 0, colSpan: 4, rowSpan: 1 },
+                { id: 'users-widget', col: 0, row: 4, colSpan: 4, rowSpan: 1 },
+                { id: 'alert-widget', col: 0, row: 3, colSpan: 4, rowSpan: 1 }
+            ];
+
+            expect(JSON.parse(await page.getLayoutOutput())).toEqual(expectedLayout);
+        });
+
+        it('should allow the rowSpan to remain the same size in stacked mode as regular mode', async () => {

```

Comment:
This test seems to be duplicated four times, is that correct?

Reply:
Good catch, I didn't notice that when I was moving them inside the stacked mode tests. They seem to be identical so I've removed them